README.md

```diff
@@ -155,6 +154,16 @@ tokenManager: {
 }
 ```
 
+Renewing tokens slightly early helps ensure a stable user experience. By default, the `expired` event will fire 30 seconds before actual expiration time. If `autoRenew` is set to true, tokens will be renewed within 30 seconds of expiration. You can customize this value by setting the `expireEarly` option. The value should be large enough to account for network latency between the client and Okta's servers.
```

Comment:
👍 I like the new wording.  Detail to clarify: 

> If `autoRenew` is set to true, tokens will be renewed within 30 seconds of expiration.

edit:

> If `autoRenew` is set to true, tokens will be renewed within 30 seconds of expiration, if accessed with `tokenManager.get()`.  

I know it seems like duplication of information, but I want to make sure we're super clear that we don't renew on  automatically on a timer.